pages/account/index.vue

```diff
@@ -116,8 +116,7 @@
     },
 
     async fetch() {
-      this.fetchLikes();
-      await this.$store.dispatch('set/fetchCreations');
```

Comment:
by removing this here I believe there's the possibility that the 'public' & 'private' tabs will show "You haven't created any x collections yet" while they have not yet been retrieved, as these are conditional based on this fetch state.

I can't actually see this because you still have to select them, but I imagine if we were to allow a user to load the account page by use of an anchor tag like /account#private they may see the message show briefly.

This is what @spyrosbek was attempting to solve previously. Spyros are you able to reproduce the issue on this branch?

Reply:
@lbiedinger Yes, at this state of the branch, I'm seeing again the "You haven't created any x collections yet" message briefly, until creations are finished fetching.